### DISCUSSION TITLE
Did this package support Mac M1?

### DISCUSSION THREAD

**Discussion by ladiesman1987** [mozman/ezdxf#407] [Q&A]
I just bought a new Mac mini device, so I wanna ask if the package has been supported new M1?

**Comment by mozman** [comment]
Yes, if Python 3.6 runs on your system. Ezdxf is designed to work also without the C-extension. If there are problems with the C-extensions, you can deliberately disable the C-extension by setting the environment variable `EZDXF_DISABLE_C_EXT=1`.

**Comment by mozman** [comment]
First beta release with binary wheels for Apple silicon: https://pypi.org/project/ezdxf/0.17.2b4/#files

**Comment by ladiesman1987** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
